### ISSUE TITLE
NGROK

### ISSUE THREAD

**Issue by yasin-rafeeque** [KasRoudra/Local2Internet#2]
Hi,
after I runned this program it was working super good but then I noticed my ngrok was replaced after using ,please see the image below and please help me to delete this ngrok

![image](https://user-images.githubusercontent.com/96828518/189084565-f6d7c02f-521a-4c31-82e0-62c7d78015a3.png)

**Comment by yasin-rafeeque** [comment]
I mean i cant even know whose account am using....

**Comment by KasRoudra** [comment]
@thizYa5R By which command you run that ngrok? I need to know that which ngrok you are using, your one or my one

**Comment by yasin-rafeeque** [comment]
@KasRoudra I really don't know but its fine , cause I just deleted  Kali Linux I was running on wsl to see which is better vm or wsl , since this problem was on wsl  there is no more problem....
You can just close the issue but I still recommend u to check if it uses ur ngrok account.....

**Comment by KasRoudra** [comment]
@thizYa5R Not at all. The term `inconshreveable` refers to the owner/developer of ngrok
